supporting_content/attributes/datetime_requirements.md

```diff
@@ -0,0 +1,37 @@
+# Datetime Requirements
+
+## Example provider mappings
+
+Current column mappings found in available data sets:
+
+| Provider  | Data set                 | Column                                               |
+|:----------|:-------------------------|:-----------------------------------------------------|
+| AWS       | CUR                      | lineItem/UsageStartDate, lineItem/UsageEndDate, etc. |
+| GCP       | Big Query Billing Export | usage_start_time, usage_end_time, etc.               |
```

Comment:
export_time: A processing time associated with an append of Cloud Billing data. This will always increase with each new export.

https://cloud.google.com/billing/docs/how-to/export-data-bigquery-tables/standard-usage

Reply:
the idea was to list a couple of examples that we map to the dimensions covered by the v0.5 scope. export_time, requires inclusion of fractions of seconds. at some point we might have to include datetime-interval format as well. we will update the specification as the need arises...

Reply:
As agreed at the squad 2 meeting, for v5.0, only the second level precision format was retained in the specification. note related to other levels of precision (minutes, milliseconds, or microseconds) added to supporting content. We plan to create an additional issue on this topic.